### PR TITLE
Fix `g_km` bug when multiple estimates are made at max time

### LINKED ISSUE #1279: [Bug Report] `g_km` fails with multiple max time records
When multiple records have estimates at max time, `g_km` fails with error:

```
Error in yval == “Survival” && data$estimate[data$time == max_time2] > : length = 2’ in coercion to ‘logical(1)’
```

### PR DESCRIPTION
**Pull request by edelarua**
# Pull Request

Fixes #1279 

Bug reported on Overflow but not able to replicate it.

### CODE REVIEW

File: R/g_km.R [hunk 1]
```diff
@@ -378,8 +378,8 @@ g_km <- function(df,
 
     y_rng <- ylim[2] - ylim[1]
 
-    if (yval == "Survival" && data$estimate[data$time == max_time2] > ylim[1] + 0.09 * y_rng &&
-      data$estimate[data$time == max_time2] < ylim[1] + 0.5 * y_rng) { # nolint
+    if (yval == "Survival" && all(data$estimate[data$time == max_time2] > ylim[1] + 0.09 * y_rng) &&
+      all(data$estimate[data$time == max_time2] < ylim[1] + 0.5 * y_rng)) { # nolint
```
Comment (Melkiades) [inline R/g_km.R:382]: it seems a good fix! I think we need to ask the roverflow requester to test it

**Review by Melkiades — APPROVED** [review]
Unblocking this if the user does not add a review ;) Thanks Emily!!!

**Comment by shajoezhu** [comment]
lets give this another day, i just pinged the user earlier today

**Comment by shajoezhu** [comment]
hi @edelarua , this is good to go. user was happy with outcome. I think we need to add some test for this. something like snapshot, but not the picture itself, but ggplot datastucture.

**Pull request merged**